spec/index.html

```diff
@@ -3499,9 +3531,19 @@ <h2>A Canonical form of N-Quads</h2>
       and <code>graphLabel</code>,
       each of which MUST be a single space (<code>U+0020</code>).</li>
     <li><a data-cite="RDF11-CONCEPTS#dfn-literal">Literals</a> with the
-      datatype <code>http://www.w3.org/2001/XMLSchema#string</code>
+      datatypes <code>http://www.w3.org/2001/XMLSchema#string</code> and
+      <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code>
```

Comment:
This is unnecessary per EBNF grammar.

```suggestion
      datatype <code>http://www.w3.org/2001/XMLSchema#string</code>
```